assets/components/contributionSelection/contributionSelectionActions.js

```diff
@@ -0,0 +1,32 @@
+// @flow
```

Comment:
Is it possible to add tests for this file?

Reply:
Done.